database/migrationsPG/20191217191729_create-users-table-pg.js

```diff
@@ -81,11 +82,12 @@ exports.up = function (knex, Promise) {
         .onDelete("CASCADE")
         .onUpdate("CASCADE");
       washer.boolean("workStatus").defaultTo(false);
-      washer.integer("rateSmall");
-      washer.integer("rateMedium").notNullable();
-      washer.integer("rateLarge");
+      washer.decimal("rateSmall", 8, 2);
```

Comment:
good work figuring out that a float would not work here and a decimal solved the issue